### ISSUE TITLE
Could not set unknown property 'enabled' for object of type dev.arunkumar.scabbard.gradle.DefaultScabbardSpec.

### ISSUE THREAD

**Issue by ursusursus** [arunkumar9t2/scabbard#10]
```
buildscript {
    repositories {
        maven {
            url "https://plugins.gradle.org/m2/"
        }
    }
    dependencies {
        classpath "gradle.plugin.dev.arunkumar:scabbard-gradle-plugin:0.1.0"
    }
}

apply plugin: "com.android.application"
apply plugin: "kotlin-android"
apply plugin: "kotlin-kapt"
apply plugin: "scabbard.gradle"

android {
    compileSdkVersion compileSdk

    defaultConfig {
        applicationId "sk.dagger.playground"
        minSdkVersion minSdk
        targetSdkVersion targetSdk
        versionCode appVersionCode
        versionName appVersionName
        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }

    signingConfigs {
        release {
            def signingFolder = "../signing"
            def propertiesFile = file("${signingFolder}/signing.properties")
            if (propertiesFile.exists()) {
                def props = new Properties()
                props.load(new FileInputStream(propertiesFile))

                keyAlias props["keyAlias"]
                keyPassword props["keyPassword"]
                storeFile file("${signingFolder}/${props["storeFile"]}")
                storePassword props["storePassword"]
            }
        }
    }

    buildTypes {
        debug {
        }
        release {
            minifyEnabled true
            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
            signingConfig signingConfigs.release
        }
    }

}

scabbard {
    enabled = true
}

dependencies {
    implementation deps.androidx.appCompat
    implementation deps.androidx.preferences
    implementation deps.androidx.recyclerView
    implementation deps.material

    implementation deps.dagger.runtime
    kapt deps.dagger.compiler

    implementation deps.rx.java
    implementation deps.rx.android
    implementation deps.rx.kotlin

    implementation deps.okhttp.client
    implementation deps.moshi.client
    implementation deps.retrofit.client
    implementation deps.retrofit.rxjavaAdapter
    implementation deps.retrofit.moshiConverter

    implementation deps.glide.client

    implementation deps.conductor.runtime
    implementation deps.jsr310
    implementation deps.kotlin

    implementation project(":features:invite")
    implementation project(":features:chat")
    implementation project(":libs:gallery")
    implementation project(":libs:giphy")
    implementation project(":base")

    testImplementation deps.test.junit

    androidTestImplementation "androidx.test:core:1.2.0"
    androidTestImplementation "androidx.test:runner:1.2.0"
    androidTestImplementation "androidx.test:rules:1.2.0"
    androidTestImplementation "androidx.test.espresso:espresso-core:3.2.0"

}
```

**Comment by bagza** [comment]
Just don't use the '=' sign, call it 
```
scabbard {
    enabled true
}
```
or
```
scabbard {
    enabled(true)
}
```

https://github.com/arunkumar9t2/scabbard/blob/f7dda39f8772285a02c680ee76be3a11d25897a1/scabbard-gradle-plugin/src/main/kotlin/dev/arunkumar/scabbard/gradle/DefaultScabbardSpec.kt#L21

**Comment by matthewrkula** [comment]
Should update the [docs](https://arunkumar9t2.github.io/scabbard/#gradle-plugin) then.

**Comment by neugartf** [comment]
https://github.com/arunkumar9t2/scabbard/pull/11
PR opened

**Comment by arunkumar9t2** [comment]
Sorry it was a silly mistake. I have merged #11. Please use

```
scabbard {
    enabled true
}
```

I am closing this issue, please feel free to open this if you still have problems.
